libc/src/time/ctime.cpp

```diff
@@ -0,0 +1,26 @@
+//===-- Implementation of ctime function ----------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "ctime.h"
+#include "../__support/common.h"
+#include "../__support/macros/config.h"
```

Comment:
Use path relative to the `libc/` folder:
```
#include "src/__support/common.h"
...
```

Reply:
Thank you for the feedback. I have used relative path now.